redis/cache/redispipebp/monitored.go

```diff
@@ -60,19 +61,40 @@ func (s MonitoredSync) Do(ctx context.Context, cmd string, args ...interface{})
 		s.Name+".do",
 		tracing.SpanTypeOption{Type: tracing.SpanTypeClient},
 	)
+	command := extractCommand(cmd)
+	active := redisprom.ActiveRequests.With(prometheus.Labels{
+		// We don't have LblClientName, LblDatabase, LblType, and LblDeployment
```

Comment:
I think `s.Name` is client name from spec?

Reply:
it isn't. `redis_client_name` is the name passed to the redis backend to identify the client. Here `s.Name` is the "slug" used to identity the client pool.

Reply:
someone can correct me if I'm wrong but I believe what `redis_client_name` means is the same as slug (we also have another spec change to rename all `_slug` labels to `_client_name`). redis does not really have `client name` in the wire protocol.